scripts/deploy.js

```diff
@@ -20,29 +21,66 @@ async function main() {
     await deployer.getAddress()
   );
 
-  console.log("Account balance:", (await deployer.getBalance()).toString());
+  console.log(
+    "Account balance:",
+    (await deployer.provider.getBalance(deployer.address)).toString(),
+    "\n"
+  );
 
-  const Token = await ethers.getContractFactory("Token");
-  const token = await Token.deploy();
-  await token.deployed();
+  const modulePath = path.join(
+    __dirname,
+    "..",
+    "ignition",
+    "modules",
+    "Token.js"
+  );
 
-  console.log("Token address:", token.address);
+  // ignition is available in the global scope
```

Comment:
As soon as [this issue](https://github.com/NomicFoundation/hardhat-ignition/issues/703) is fixed this line should be changed